packages/acceptance-tests/pkg-tests-specs/sources/commands/workspaces/foreach.test.js

```diff
@@ -469,6 +470,72 @@ describe(`Commands`, () => {
         },
       ),
     );
+
+    // FIXME: figure out why test thinks `--since` option is invalid
+    test.skip(
```

Comment:
The tests are working for `yarn workspaces list`, but I'm having trouble getting them working here for `yarn workspaces foreach`. Each test was failing on my local saying that `--since` is an unsupported option.

If anyone has any clues on what could be causing it, I could use the help. Otherwise I'll continue trying to debug this when I get some time over the next couple days.